### PR TITLE
Update components.ex

### PR DESCRIPTION
**Pull request by cvkmohan**
eex starting `<%` missing.

### CODE REVIEW

File: priv/templates/phx.gen.live/components.ex [hunk 1]
```diff
@@ -698,9 +698,9 @@ defmodule <%= @web_namespace %>.Components do
     # should be written to the errors.po file. The :count option is
     # set by Ecto and indicates we should also apply plural rules.
     if count = opts[:count] do
-      Gettext.dngettext( @web_namespace %>.Gettext, "errors", msg, msg, count, opts)
+      Gettext.dngettext( <% @web_namespace %>.Gettext, "errors", msg, msg, count, opts)
```
Comment (mcrumm) [inline priv/templates/phx.gen.live/components.ex:701]:
```suggestion
      Gettext.dngettext(<%= @web_namespace %>.Gettext, "errors", msg, msg, count, opts)
```

File: priv/templates/phx.gen.live/components.ex [hunk 2]
```diff
@@ -698,9 +698,9 @@ defmodule <%= @web_namespace %>.Components do
...
     else
-      Gettext.dgettext( @web_namespace %>.Gettext, "errors", msg, opts)
+      Gettext.dgettext( <% @web_namespace %>.Gettext, "errors", msg, opts)
```
Comment (mcrumm) [inline priv/templates/phx.gen.live/components.ex:703]:
```suggestion
      Gettext.dgettext(<%= @web_namespace %>.Gettext, "errors", msg, opts)
```

**Review by mcrumm — COMMENTED** [review]
Thanks! It needs a couple tiny changes and then we can ship it :)

**Comment by mcrumm** [comment]
Thank you for the PR! I was in the middle of another update that includes the same fix (#4947) :)

**Comment by cvkmohan** [comment]
Oh. Thanks. Looking forward to new version

**Pull request closed without merging**